packages/web/src/components/leaving-audius-modal/LeavingAudiusModal.module.css

```diff
@@ -0,0 +1,34 @@
+.modalBody {
+  width: 480px;
+}
+
+.text {
+  font-size: var(--font-l);
+  font-weight: var(--font-medium);
+}
+
+.content > div {
```

Comment:
it's so annoying, but the `<ModalContent>` has an inner div that makes this selector necessary

Reply:
Oooh nice! Yeah passing forward seems like a great idea. But unf that would break a lot of existing modals so it'd have to be done separately

Fwiw `ModalContent` always has exactly one child div by design so `:first-child` is kinda unnecessary imo